docs/content/development/extensions-core/datasketches-aggregators.md

```diff
@@ -153,3 +162,105 @@ sample query for, How many unique users visited both product A and B?
   ]
 }
 ```
+
+## quantiles aggregator ( EXPERIMENTAL )
+This aggregator is used to compute quantiles and histograms. Only double data type (other number types would be coerced) can be used as input to this aggregator.
+
+At ingestion time, quantiles sketches would get created and stored in the segments. At query time, quantile sketches can be used to compute quantiles and histograms from that column. 
+
+Note that quantile sketches are under active development and still considered experimental.
+
+### Aggregators
+
+```json
+{
+  "type" : "datasketchesQuantilesSketch",
+  "name" : <output_name>,
+  "fieldName" : <metric_name>,
+
+  //following boolean field is optional. This should only be used at
+  //indexing time if your input data contains quantile sketch objects.
+  //that would be the case if you use datasketches library outside of Druid,
+  //say with Pig/Hive, to produce the data that you are ingesting into Druid
+  "isInputSketch": false
+
+  //following field is optional, default = 1024.
+  //it must be a power of 2 and less than 65536
+  //default value here is arbitrary and appropriate value for specific usecase
+  //should be determined. See [quantiles-accuracy](http://datasketches.github.io/docs/QuantilesAccuracy.html) .
```

Comment:
have you tried rendering these docs with jekyll? I really think the link to the accuracy should be linked outside of a comment in a json blob